### PR TITLE
Documentation: adds more docCategory tags

### PR DESCRIPTION
**Pull request by Vitalius1**
#### Pull request checklist
- [X] Include a change request file using `$ npm run change`

#### Description of changes

Adds some more docCategory tags for API items that were missing them. There are still quite a few to add.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/8940)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.base.tsx [hunk 1]
```diff
@@ -14,6 +14,9 @@ export interface IChoiceGroupState {
   keyFocused?: string | number;
 }
 
+/**
+ * {@docCategory ContextualMenu}
```
Comment (natalieethell) [inline packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.base.tsx]:
```suggestion
 * {@docCategory ChoiceGroup}
```

File: packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.base.tsx [hunk 2]
```diff
@@ -7,6 +7,9 @@ import { IProcessedStyleSet } from '../../../Styling';
 
 const getClassNames = classNamesFunction<IChoiceGroupOptionStyleProps, IChoiceGroupOptionStyles>();
 
+/**
+ * {@docCategory ContextualMenu}
```
Comment (natalieethell) [inline packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.base.tsx]:
```suggestion
 * {@docCategory ChoiceGroup}
```

File: packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.types.ts [hunk 3]
```diff
@@ -3,9 +3,19 @@ import { ITheme, IStyle } from '../../../Styling';
 import { IRefObject } from '../../../Utilities';
 import { IChoiceGroupOption } from '../../ChoiceGroup/ChoiceGroup.types';
 
+/**
+ * {@docCategory ContextualMenu}
```
Comment (natalieethell) [inline packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.types.ts]:
```suggestion
 * {@docCategory ChoiceGroup}
```

File: packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.types.ts [hunk 4]
```diff
@@ -3,9 +3,19 @@ import { ITheme, IStyle } from '../../../Styling';
...
+ */
 export type OnFocusCallback = (ev?: React.FocusEvent<HTMLElement | HTMLInputElement>, props?: IChoiceGroupOption) => void | undefined;
+
+/**
+ * {@docCategory ContextualMenu}
```
Comment (natalieethell) [inline packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.types.ts]:
```suggestion
 * {@docCategory ChoiceGroup}
```

File: packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.types.ts [hunk 5]
```diff
@@ -3,9 +3,19 @@ import { ITheme, IStyle } from '../../../Styling';
...
+ */
 export type OnChangeCallback = (evt?: React.FormEvent<HTMLElement | HTMLInputElement>, props?: IChoiceGroupOption) => void;
 
+/**
+ * {@docCategory ContextualMenu}
```
Comment (natalieethell) [inline packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.types.ts]:
```suggestion
 * {@docCategory ChoiceGroup}
```

File: packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.types.ts [hunk 6]
```diff
@@ -49,6 +59,9 @@ export interface IChoiceGroupOptionProps extends IChoiceGroupOption {
   name?: string;
 }
 
+/**
+ * {@docCategory ContextualMenu}
```
Comment (natalieethell) [inline packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.types.ts]:
```suggestion
 * {@docCategory ChoiceGroup}
```

File: packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.types.ts [hunk 7]
```diff
@@ -59,6 +72,9 @@ export interface IChoiceGroupOptionStyleProps {
   focused?: boolean;
 }
 
+/**
+ * {@docCategory ContextualMenu}
```
Comment (natalieethell) [inline packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.types.ts]:
```suggestion
 * {@docCategory ChoiceGroup}
```

**Comment by Vitalius1** [comment]
Thanks @natalieethell  for catching that. I fixed them!

**Pull request merged**